tests/js/setupFramework.ts

```diff
@@ -0,0 +1,17 @@
+/* global process */
+import '@visual-snapshot/jest';
+
+export {};
+
+declare global {
+  namespace jest {
+    interface Matchers<R> {
+      toSnapshot(): CustomMatcherResult;
+    }
+  }
+}
```

Comment:
So the `import` should grab in this type, but Cloud Build was failing (even though GHA was passing)